packages/squiggle-lang/src/rescript/Utility/E/E_R.res

```diff
@@ -1,15 +1,14 @@
 /* R for Result */
 
 exception Assertion(string)
-module U = E_U
 
 open Belt.Result
 let result = (okF, errF, r) =>
   switch r {
   | Ok(a) => okF(a)
   | Error(err) => errF(err)
   }
-let id = e => e |> result(U.id, U.id)
+let id = e => e |> result(e => e, e => e)
```

Comment:
This function is not used anywhere and doesn't seem very useful.

Reply:
Removed in #1318.